js/utils/dom.js

```diff
@@ -19,3 +19,41 @@ export function isScrolledIntoView(element) {
   // Check its within the document viewport
   return top <= document.documentElement.clientHeight;
 }
+
+export function throttle(handler, fps) {
```

Comment:
These are already both provided by [underscore](http://underscorejs.org/) which is a dependency of backbone and a lib we very heavily utilize. Can we just use those?